### PR TITLE
Add Config.FileContent

### PR DESCRIPTION
**Pull request by willfaught**
I need to be able to embed a swagger.json file into a Go program using the embed package. Config.FileContent enables that.

## Summary by CodeRabbit

- **New Features**
	- Introduced the ability to directly specify Swagger file content in configurations, enhancing flexibility.
- **Tests**
	- Added comprehensive tests for Swagger middleware configurations, including custom paths and multiple instances.

### CODE REVIEW

**Comment by willfaught** [comment]
Note that I tried to follow the way the existing code does and tests things.

File: swagger/swagger.go [hunk 1]
```diff
@@ -85,16 +91,20 @@ func New(config ...Config) fiber.Handler {
...
 
-	// Verify Swagger file exists
-	if _, err := os.Stat(cfg.FilePath); os.IsNotExist(err) {
-		panic(fmt.Errorf("%s file does not exist", cfg.FilePath))
-	}
+	rawSpec := cfg.FileContent
+	if len(rawSpec) == 0 {
+		// Verify Swagger file exists
+		_, err := os.Stat(cfg.FilePath)
+		if os.IsNotExist(err) {
```
Comment (efectn) [inline swagger/swagger.go:98]: Use `errors.Is(err, fs.ErrNotExist)` instead

Comment (willfaught) [reply swagger/swagger.go:98]: That is existing code. I tried to minimize the changes. Do you still want me to change it?

**Comment by gaby** [comment]
@willfaught Want me to take over and get this merged?

**Comment by willfaught** [comment]
I forgot I had this PR open. If a maintainer is willing to attend to this PR to get it merged, I can reopen the PR.

**Comment by gaby** [comment]
@willfaught I will take a look if you can re-open it, thanks!

**Comment by willfaught** [comment]
GitHub won't let me reopen this PR. Opened a new PR: https://github.com/gofiber/contrib/pull/1085.

**Pull request closed without merging**